engine/src/main/java/org/terasology/monitoring/gui/ChunkMonitorDisplay.java

```diff
@@ -694,12 +631,12 @@ private void renderSelectedChunk(Graphics2D g, int offsetx, int offsety, Vector3
         }
 
         private void renderBox(Graphics2D g, int offsetx, int offsety, Rectangle box) {
-            g.setColor(Color.white);
+            g.setColor(Color.WHITE);
```

Comment:
This case change on the other hand is definitely good 👍

Reply:
Thanks! :smiley: